### DISCUSSION TITLE
Bookmarks

### DISCUSSION THREAD

**Discussion by AdamGr1984** [sumatrapdfreader/sumatrapdf#3250] [Q&A] [answered]
Hi,
I can't get a Bookmark window to open. I tried with a few documents, but it's still grey. 
![image](https://user-images.githubusercontent.com/124278059/216356267-7de43dc2-7e75-4cf5-976c-652876ac4492.png)

Adam

**Comment by GitHubRulesOK** [comment, marked as answer]
Bookmarks (Table Outline of Contents) are very SHORT word hyper links, added by the Author inside the Document, SumatraPDF can only show them if present. The addition is an editing function and they are not part of annotation (but in some editors they can be added as hyperlinks) PDF has several types of Links such as Page GoTo or Named Destinations that are not considered part of the Outline, so do not show, but may include an Outline entry.
